src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationLoaderSecurity7.java

```diff
@@ -131,7 +131,7 @@ public void noData(String id, String type) {
 
                 //when index was created with ES 6 there are no separate tenants. So we load just empty ones.
                 //when index was created with ES 7 and type not "security" (ES 6 type) there are no rolemappings anymore.
-                if(cs.state().metadata().index(securityIndex).getCreationVersion().before(Version.V_7_0_0) || "security".equals(type)) {
```

Comment:
same as above

Reply:
No, it's not used anywhere else in this file.